### PR TITLE
add support for omnibar signOutRedirectUrl

### PR DESCRIPTION
**Pull request by Blackbaud-JackieKirchhoff**

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -124,7 +124,8 @@ export class AppComponent implements OnInit {
     if (omnibarBootstrapConfig) {
       const omnibarConfig: BBOmnibarConfig = {
         serviceName: omnibarBootstrapConfig.serviceName,
-        experimental: omnibarBootstrapConfig.experimental
+        experimental: omnibarBootstrapConfig.experimental,
+        signOutRedirectUrl: omnibarBootstrapConfig.signOutRedirectUrl
```
Comment (Blackbaud-SteveBrush) [inline src/app/app.component.ts]: @Blackbaud-JackieKirchhoff: You'll also need to update the `BBOmnibarConfig` interface, located at [`blackbaud/auth-client`](https://github.com/blackbaud/auth-client/blob/master/src/omnibar/omnibar-config.ts).

Comment (Blackbaud-JackieKirchhoff) [inline src/app/app.component.ts]: I have a PR out to update BBOmnibarConfig

**Comment by Blackbaud-BobbyEarl** [comment]
With #145 and #149 this will no longer be necessary.

**Comment by Blackbaud-JackieKirchhoff** [comment]
This is no longer necessary, closing.

**Pull request closed without merging**